lib/datastore/index.js

```diff
@@ -192,19 +192,41 @@ var util = require('../common/util.js');
  * //-
  * // <h3>Paginating Records</h3>
  * //
- * // By default, all records are returned at once. To control pagination, you
- * // can disable this on your query with {module:datastore/query#autoPaginate}.
+ * // Imagine building a website that allows a user to sift through hundreds of
+ * // their contacts. You'll likely want to only display a subset of these at
+ * // once, so you set a limit.
  * //-
- * query.autoPaginate(false);
+ * var express = require('express');
+ * var app = express();
  *
- * datastore.runQuery(query, function(err, entities, nextQuery) {
- *   // entities = [...];
+ * var NUM_RESULTS_PER_PAGE = 15;
  *
- *   if (nextQuery) {
- *     // Run `datastore.runQuery` again with the prepared query to retrieve the
- *     // next set of results.
- *     datastore.runQuery(nextQuery, function(err, entities, nextQuery) {});
+ * app.get('/contacts', function(req, res) {
+ *   var query = datastore.createQuery('Contacts')
+ *     .limit(NUM_RESULTS_PER_PAGE);
+ *
+ *   if (req.query.nextPageCursor) {
+ *     query.start(req.query.nextPageCursor);
  *   }
+ *
+ *   datastore.runQuery(query, function(err, entities, info) {
+ *     if (err) {
+ *       // Error handling omitted...
+ *       return;
+ *     }
+ *
+ *     // Respond to the front end with the contacts and the cursoring token
+ *     // from the query we just ran.
+ *     var frontEndResponse = {
+ *       contacts: entities
+ *     };
+ *
+ *     if (info.moreResults !== 'NO_MORE_RESULTS') {
```

Comment:
WDYT about moving this to some kind of static-y constant-y value (as well as any other values here)?

```js
info.moreResults === datastore.NO_MORE_RESULTS
``